tensorboard/components/experimental/plugin_util/message.ts

```diff
@@ -80,23 +83,23 @@ namespace tb_plugin.lib.DO_NOT_USE_INTERNAL {
         return;
       }
 
-      let payload = null;
-      let error = null;
-      if (this.listeners.has(message.type)) {
-        const callback = this.listeners.get(message.type);
+      let replyPayload = null;
+      let replyError = null;
+      if (this.listeners.has(type)) {
+        const callback = this.listeners.get(type) as MessageCallback;
         try {
-          const result = await callback(message.payload);
-          payload = result;
+          const result = await callback(payload);
+          replyPayload = result;
         } catch (e) {
-          error = e;
+          replyError = e;
         }
       }
       const replyMessage: Message = {
-        type: message.type,
-        id: message.id,
-        payload,
-        error,
-        isReply: true,
+        ['type']: type,
```

Comment:
Here and above, Closure compiler is not properly typed and JSCompiler eagerly mangle these properties. In general, that is fine but since we are creating two separate bundles (one for tensorboard.html and one for the plugin_lib), the mangled name has to be stable (or across different versions of the plugin_lib).